tests/api/test_community_events.py

```diff
@@ -0,0 +1,38 @@
+import datetime
+import pickle
+import logging
+
+from actions.api import community_events
+
+
+def test_get_community_events(caplog):
+    """
+    Test that format of current events page is as expected and that parsing of a known events page finds the right events
+    """
+    with caplog.at_level(logging.WARNING):
+        community_events.get_community_events()
+    assert "Error when trying to parse event details from html." not in caplog.text
+
+
+def test_parse_community_events(mocker):
+    mocker.patch.object(
+        community_events,
+        "get_community_page",
+        return_value=pickle.load(open("tests/data/events_page.pkl", "rb")),
+    )
```

Comment:
To be safe, I think we want to do this in a `with open`, so something like
```suggestion
    with open("tests/data/events_page.pkl", "rb") as events_page: 
        mocker.patch.object(
            community_events,
            "get_community_page",
            return_value=pickle.load(events_page),
        )
```